bookkeeper-server/pom.xml

```diff
@@ -112,6 +112,16 @@
       <groupId>net.java.dev.jna</groupId>
       <artifactId>jna</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.bookkeeper.stats</groupId>
+      <artifactId>prometheus-metrics-provider</artifactId>
```

Comment:
these are test dependencies, right?

Reply:
moved it to testing dependencies with test scope